tensorflow/lite/micro/kernels/lstm_eval.h

```diff
@@ -29,6 +29,196 @@ limitations under the License.
 #include "tensorflow/lite/micro/micro_log.h"
 
 namespace tflite {
+
+// Interface to access all the TempTfLiteTensors of the LSTM kernel during the
+// preparation phase. Can only be constructed through the constructor to avoid
+// memory leakage. All TempTfLiteTensors will be deallocated through the
+// destructor.
+class LstmTensors {
```

Comment:
can you move the implementation of this class into the cc file?

Reply:
Done